src/main/resources/default/assets/common/common-polyfills.js.pasta

```diff
@@ -179,3 +179,21 @@ if (typeof Object.assign != 'function') {
         configurable: true
     });
 }
+
+/**@
+* Polyfill for the Object.entries function
+* see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#polyfill
+*/
+if (!Object.entries) {
+    Object.entries = function(obj) {
+        var ownProps = Object.keys(obj),
+            i = ownProps.length,
+            resArray = new Array(i); // preallocate the Array
```

Comment:
```suggestion
        const keys = Object.keys(obj);
        const result = new Array(keys.length);
        
        let index = keys.length;
```

Reply:
I'd prefer let/const over var....

Reply:
The current usage of var enables us to use Object.keys in IE versions > 8. If we use const / let, we'd only support IE versions > 10 (effectively only 11). So the question is: Would it be enough to only support IE11?

Reply:
yep